scripts/format_new.py

```
@@ -0,0 +1,170 @@
+    def save_contact(self, contact: Contact) -> None:
+        """Save contact to store."""
+        self.contacts[contact.name] = {
+            "phone": contact.phone,
+            "emails": contact.emails
+        }
+        
+        with open(self.file_path, "w") as f:
+            json.dump(self.contacts, f, indent=2)
+
+
+def get_contact_info(name: str) -> Optional[Contact]:
```

Ruff (0.8.2)

Use `X | Y` for type annotations

Convert to `X | Y`

(UP007)